pkg/controller/controller.go

```diff
@@ -291,14 +306,16 @@ func (c *Controller) loadConfig(path string) {
 	logger.V(3).Info("updated config", "path", path, "config", c.config)
 
 	// requeue all clusters
-	lister := typed.ListerFor[*v1alpha1.SpiceDBCluster](c.Registry, typed.NewRegistryKey(OwnedFactoryKey, v1alpha1ClusterGVR))
-	clusters, err := lister.List(labels.Everything())
-	if err != nil {
-		utilruntime.HandleError(err)
-		return
-	}
-	for _, cluster := range clusters {
-		c.enqueue(v1alpha1ClusterGVR, cluster)
+	for _, ns := range c.namespaces {
```

Comment:
if the namespace list is empty, where does it get watched then?

Reply:
`namespaces` gets defaulted to `[]string{corev1.AllNamespace}` (which is just the empty string) if it is empty on startup.